apps/fluent-tester/tsconfig.json

```diff
@@ -4,13 +4,10 @@
     "baseUrl": ".",
     "outDir": "lib",
     "paths": {
-      "*": [
-        "*",
-        "*.win32"
-      ]
-    }
+      "*": ["*", "*.win32", "./*"],
+      "src/*": ["./src/*", "src"]
+    },
+    "types": ["@wdio/sync", "@wdio/jasmine-framework", "node", "@types/jasmine"]
   },
-  "include": [
-    "src"
-  ]
-}
\ No newline at end of file
+  "include": ["./src/**/*.ts"]
```

Comment:
interesting, do you remember what problems you had such that you needed to change the include and paths?

Reply:
I would have also thought that the types option wouldn't be necessary. I thought types got auto included.

Reply:
At the time, I was running into a lot of TypeScript errors. This is the exact syntax on the WebDriverIO website for Typescript implementation, so I followed it. However, I believe I can change it back without any difference. Testing now.

Reply:
Changed back in newest iteration